### PR TITLE
ci: trim job permissions

### PR DESCRIPTION
**Pull request by viceice**
## Changes:

Trim GitHub token permissions for jobs.

## Context:
 see #10794

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Comment by HonkingGoose** [comment]
@viceice The link to the issue is not working (missing a `#` character).

File: .github/workflows/label-actions.yml [hunk 1]
```diff
@@ -4,6 +4,9 @@ on:
   issues:
     types: [labeled]
 
+permissions:
```
Comment (HonkingGoose) [inline .github/workflows/label-actions.yml:7]: The example config from `dessant/label-actions` in their README gives these permissions.
See https://github.com/dessant/label-actions#examples

```yml
name: 'Label Actions'

on:
  issues:
    types: [labeled, unlabeled]
  pull_request:
    types: [labeled, unlabeled]

permissions:
  contents: read
  issues: write
  pull-requests: write

jobs:
  action:
    runs-on: ubuntu-latest
    steps:
      - uses: dessant/label-actions@v2
 ```

We don't process pull requests so we should NOT give it any `pull-requests` permission. 😄 

But maybe the action does need the `contents: read` permission, not sure about that.

https://docs.github.com/en/rest/reference/permissions-required-for-github-apps#permission-on-contents

File: .github/workflows/label-actions.yml [hunk 2]
```diff
@@ -4,6 +4,9 @@ on:
...
+  issues: write
```
Comment (viceice) [inline .github/workflows/label-actions.yml:8]:
```suggestion
  contents: read
  issues: write
```

Ok, adding `contents: read`

**Review by HonkingGoose — APPROVED** [review]
I think this looks OK, but we should check the logs after merging this PR, to see if it works correctly.

Also do we intend to restrict official actions as well, like `actions/setup-node` and so?

**Review by rarkins — APPROVED** [review]

**Pull request merged**